engine/src/main/java/org/terasology/rendering/nui/layers/ingame/OnlinePlayersOverlay.java

```diff
@@ -50,7 +52,8 @@ private String determinePlayerListText() {
                 sb.append("\n");
             }
             ClientComponent clientComp = clientEntity.getComponent(ClientComponent.class);
-            sb.append(PlayerUtil.getColoredPlayerName(clientComp.clientInfo));
+            sb.append(PlayerUtil.getColoredPlayerName(clientComp.clientInfo) +
```

Comment:
No real need to line break here. It is only 138 characters, well within our code style max line length.

Better yet might be fixing the pre-existing inspection warning over using String concatenation with a StringBuffer (could use two append calls). Also missing a space between `"` and `+` (super tiny issue)